cpp/src/merge/merge.cu

```diff
@@ -335,11 +338,7 @@ struct column_merger
     if (lcol.has_nulls() || rcol.has_nulls())
```

Comment:
Not this line but the enclosing function is sfinae enabled for not is_fixed_width. This will be wrong after dictionary columns.

Reply:
Good catch. These will be tricky to find. 
This probably does not need to be sfinae. What do you think?

Reply:
With template specialisations in general, you have the issue:
If you use default template for all except strings and dictionary types, any new type added will silently fail.
If you use default template for failure, then you'd have to specialise for each fixed width type. Makes for messy, verbose code.